__tests__/components/sections/communities/overview/scoreboard/index.test.tsx

```diff
@@ -0,0 +1,50 @@
+import "@testing-library/jest-dom";
+import { fireEvent, screen } from "@testing-library/react";
+import { renderWithRedux } from "@__mocks__/renderWithRedux";
+import ScoreboardOverview from "@/components/sections/communities/overview/scoreboard";
+import { mockScoreboard } from "@__mocks__/scoreboard";
+
+jest.mock("next/router", () => ({
+  useRouter: () => ({
+    pathname: "next",
+  }),
+}));
+describe("ScoreboardOverview", () => {
+  let items = 3;
+
+  const loadMore = () => {
+    items = items + 10;
+  };
+
+  it("displays the scoreboard overview", () => {
+    renderWithRedux(<ScoreboardOverview testId="scoreboardId" />, {
+      scoreboard: { list: [mockScoreboard], loading: true, filterBy: "" },
+    });
+    expect(screen.getByTestId("scoreboardId")).toBeInTheDocument();
+  });
+
+  it("renders scoreboard cards when not loading and list is not empty", () => {
+    renderWithRedux(<ScoreboardOverview />, {
+      scoreboard: { list: [mockScoreboard], loading: false, filterBy: "" },
+    });
+    if ([mockScoreboard] && [mockScoreboard].length !== 0) {
+      [mockScoreboard].slice(0, items).forEach((item) => {
+        expect(screen.getByText(item.user.displayName)).toBeInTheDocument();
```

Comment:
```suggestion
        expect(screen.getByText(item.user.displayName)).toBeInTheDocument();
```
you can add another test to check if the text is as expected